tests/tests/swfs/visual/filters/displacement_map/Test.as

```diff
@@ -0,0 +1,69 @@
+﻿package  {
+	
+	import flash.display.MovieClip;
+	import flash.display.BitmapData;
+	import flash.display.Bitmap;
+	import flash.display.BitmapDataChannel;
+	import flash.filters.DisplacementMapFilter;
+    import flash.filters.DisplacementMapFilterMode;
```

Comment:
Oh no, my OCD... (Four spaces instead of a tab. But whatever.)

Reply:
Blame flash's editor <_<